meta-iot2000-example/recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -1,7 +1,5 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/swupdate:"
 
-#SRCREV = ""
-
```

Comment:
This belong into the previous patch (squash).

Reply:
Done. Sorry for the mistake, was not intended.